src/exchange/account/delegation/account-delegation.controller.js

```diff
@@ -16,11 +16,13 @@ angular.module('Module.exchange.controllers').controller(
       this.$routerParams = this.services.Exchange.getParams();
       this.currentAccount = this.services.navigation.currentActionData.primaryEmailAddress;
       this.searchValue = null;
-
+      this.selectedDomain = this.services.navigation.currentActionData.completeDomain;
+      this.allDomainsOption = { displayName: this.services.$translate.instant('exchange_all_domains'), name: '' };
+      this.availableDomains = [this.allDomainsOption].concat(this.services.navigation.currentActionData.availableDomains);
```

Comment:
You will ended up with an ESLint `max-len` error.

Reply:
fixed